Resources/Prototypes/_Mono/Shipyard/Expedition/picses.yml

```
@@ -0,0 +1,40 @@
+  stations:
+    Picses:
+      stationProto: StandardFrontierExpeditionVessel
+      components:
+        - type: StationNameSetup
+          mapNameTemplate: 'Picses {1}'
+          nameGenerator:
+            !type:NanotrasenNameGenerator
+            prefixCreator: '14'
+        - type: StationJobs
+          availableJobs:
+            Contractor: [ 0, 0 ]
```

YAMLlint (1.35.1)

[error] too many spaces inside brackets

(brackets)